app/services/EventReportService.scala

```diff
@@ -168,25 +281,39 @@ class EventReportService @Inject()(eventReportConnector: EventReportConnector,
   def getEventSummary(pstr: String, version: String, startDate: String)
                      (implicit headerCarrier: HeaderCarrier, ec: ExecutionContext): Future[JsArray] = {
 
+    def replaceWindup(jsArray: JsArray) = Json.toJson(jsArray.value.map { jsValue =>
+      if (jsValue.as[String] == "0") "WindUp"
```

Comment:
I think the `WithName` property of WindUp is now no longer `0` but is actually just `WindUp` in main following the changes required for the WindUp audit. This check might need slight modification

See here: https://github.com/hmrc/pension-scheme-event-reporting/blob/main/app/models/enumeration/EventType.scala#L27